src/main/Main.hs

```diff
@@ -1173,15 +1173,19 @@ withMiniConfigAndLock go inner =
 -- | Project initialization
 initCmd :: InitOpts -> GlobalOpts -> IO ()
 initCmd initOpts go = do
-    pwd <- getCurrentDir
-    withMiniConfigAndLock go (initProject pwd initOpts (globalResolver go))
+    let selectedDirs = searchDirs initOpts
```

Comment:
This file can be reverted. We are already passing the dirs to initProject as part of initOpts. We can extract this information there itself rather than passing it separately.